Tests/FrameworkTests/Satellite Tests/Text Fields Tests/SSNTextFieldTests.swift

```diff
@@ -56,12 +56,12 @@ class SSNTextFieldTests: VGSCollectBaseTestCase {
       ssnTextField.configuration = config
       
       let notValidSSN = [
-        "111111111", "222222222", "555555555",
```

Comment:
111111111 is not SSN, why it's removed

Reply:
Hi @dmytrokhl, I'm using the same validation we are using in Android, that way we use the same values to test around all platforms.
Also, with the new regex, 111111111 is valid, I will ask in the standup meeting with Anna and Dima tomorrow